app/views/notifications/lists.turbo_stream.erb

```diff
@@ -0,0 +1,3 @@
+<% turbo_stream.update 'notifications' do %>
```

Comment:
I think this is not in use, but need to think the case, when this types of files are in use.